recipe/build.sh

```diff
@@ -1,15 +1,78 @@
-#!/usr/bin/env bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* .
+ #!/usr/bin/env bash
+set -ex
 
-# use pkg-config from BUILD_PREFIX
-export PKG_CONFIG=$BUILD_PREFIX/bin/pkg-config
+if [ "${CONDA_BUILD_CROSS_COMPILATION}" = "1" ]; then
+    unset _CONDA_PYTHON_SYSCONFIGDATA_NAME
+    (
+        mkdir -p native-build
+
+        export CC=$CC_FOR_BUILD
+        export CXX=$CXX_FOR_BUILD
+        export AR=$($CC_FOR_BUILD -print-prog-name=ar)
+        export NM=$($CC_FOR_BUILD -print-prog-name=nm)
+        export LDFLAGS=${LDFLAGS//$PREFIX/$BUILD_PREFIX}
+        export PKG_CONFIG_PATH=${BUILD_PREFIX}/lib/pkgconfig
+
+        # Unset them as we're ok with builds that are either slow or non-portable
+        unset CFLAGS
+        unset CPPFLAGS
+        unset CXXFLAGS
+
+        meson setup native-build --prefix="${BUILD_PREFIX}" -Dlibdir=lib
+        # This script would generate the functions.txt and dump.xml and save them
+        # This is loaded in the native build. We assume that the functions exported
+        # by glib are the same for the native and cross builds
+        export GI_CROSS_LAUNCHER=$BUILD_PREFIX/libexec/gi-cross-launcher-save.sh
+        ninja -C native-build -j ${CPU_COUNT}
```

Comment:
Looks like a missing install here.
```suggestion
        ninja -C native-build -j ${CPU_COUNT}
        ninja -C native-build install -j ${CPU_COUNT}
```

See: https://github.com/conda-forge/gdk-pixbuf-feedstock/blob/30a0a914ec99dd0ab03f6a694e8564ee5709ed9d/recipe/build.sh#L69